code/datums/components/itempicky.dm

```diff
@@ -0,0 +1,35 @@
+/// You can only hold whitelisted items
```

Comment:
can a human use the give command to give you an item, like what used to be done to bypass the xenomorph held item restrictions?

Reply:
I can't find a give command

Reply:
Give only attempts on carbon mobs, and drones are not carbons. Even if it did work, it uses the `put_in_hands` proc, so I forced that on the drone, and it fails when it's not on the whitelist. So this works correctly